### PR TITLE
Antag rebalance

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

This PR implementing bunch of changes described in [antag rebalance doc](https://docs.google.com/document/d/1FeYkH2MXWMUCK45zPyepfPL6ub8RNr_DYqQ7fb1qlCs).
Changed so far:

In uplink,

"Concealed Cane Sword" moved to "Gentlemen's Kit" in devices and tools category, with "new" insulated white gloves.
Cane was also made into tool. Concealed sword have base damage of 20. Both can hold 5 tool mods. Costs 3 TC.

"Subversive Soap" moved to "Crime Scene Cleanup Kit" in stealth and camouflage items category, along with cleaner grenade, expanded body bag (takes two bodies instead of one) and bottle of space cleaner.
Soap can only clean whole tile, not individual decals. Stun from foam halved. Costs 2 TC.

"Chameleon Headset" added to "Chameleon Kit", it comes with no encryption keys.

"Assassin's Dagger" removed.

"Raw Telecrystals" uplink spawn option removed.

"Uplink Unit" uplink spawn option get 80% more TC on start.

UPD:

Fixed typo in cracker and frag grenade box order. Now it properly states that box contain 4 grenades.

"Energy Blade Projector" added to hardsuit modules category. It got stun mode, modes switched by activating blade in hands. Stun blade on par with stun baton, but ignores armor and consumes a lot of power. Also no more darts. Costs 5 TC.
image

"Spy Bug" got special interactions with vendomats, consoles, fuel/water tanks and chairs/stools. When clicked on any of those - bug would be placed under that object.

"Toxin Kit" now have six vials instead of one, amatoxin and potassium chloride replaced space drugs and mind breaker as a potential vial content.

"Explosive implant" have been reworked.
It bounds to uplink from which it was ordered, and could be controlled from new "Implants Control" tab.
From there, implants could be detonated, extracted (same as being rejected by cruciform, but implant keep functioning after) or configured. Configuration includes activation phrase, activation on wearer's death and delay of after death explosion, if such enabled.

image

image

UPD:
All mentions of traitor purged, only contractor left now. Probably would have to correct ban lists.

~To be continued.~

## Why It's Good For The Game

Why not?

## Changelog
:cl:
add: Added new things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
imageadd: added some icons and images
code: changed some code
/:cl:

### CODE REVIEW

**Comment by zombiemimic** [comment]
why remove the raw TC?

**Comment by SirRichardFrancis** [comment]
> why remove the raw TC?

Read the doc. If you still have questions after reading it - contact FortuneMaster#5544

**Comment by zombiemimic** [comment]
Ah, I thought you were removing the possibility to remove TC

File: code/datums/uplink/devices and tools.dm [hunk 1]
```diff
@@ -10,6 +10,11 @@
 	path = /obj/item/storage/toolbox/syndicate
 	desc = "Danger. Very robust. Filled with advanced tools."
 
+/datum/uplink_item/item/tools/gentlemen_kit
+	name = "Gentlemen's Kit"
```
Comment (Firefox13) [inline code/datums/uplink/devices and tools.dm:14]: `Gentlemen` is plural, `gentleman` is singular. Pretty sure you meant second one?

Comment (FortuneMaster) [reply code/datums/uplink/devices and tools.dm:14]: my mistake, should be gentelman

**Comment by SirRichardFrancis** [comment]
Let's be done with it until conflicts pile up.

**Comment by SirRichardFrancis** [comment]
Just downloaded cev_eris.dme from master branch and after 2 lines change it conflicts. Can't help it.

**Comment by SirRichardFrancis** [comment]
Пиздец

**Pull request closed without merging**